iocage/lib/Jail.py

```diff
@@ -410,7 +410,9 @@ def start(
         exec_prestart: typing.List[str] = []
         exec_start: typing.List[str] = []
         exec_started: typing.List[str] = [
-            f"echo \"export IOCAGE_JID=$IOCAGE_JID\" > {self.script_env_path}"
+            f"echo \"export IOCAGE_JID=$IOCAGE_JID\" > {self.script_env_path}",
+            "set -e",
+            "set -u",
```

Comment:
this can be just one line `set -eu` but we should maybe just have it… uhm…automatically written… elsewhere.